src/components/artemis/documentation/index.md

```diff
@@ -1,7 +1,31 @@
 ---
-layout: iframe
-title: Apache ActiveMQ Artemis Documentation
-iframe: docs
+layout: default_md
+title: ActiveMQ Artemis Documentation
 title-class: page-title-artemis
+type: artemis
 ---
 
+The user documentation is available in HTML and PDF formats, select the relevant version to view or download.
+
+#### Latest User Documentation:
+
+*   [HTML](latest)
+*   [PDF](latest/book.pdf)
+
+The user documentation for earlier versions are also available on the <a href="../download/past_releases" target="_parent">past releases</a> page.
+
+#### Examples:
+
+Examples can be found in the <a href="https://github.com/apache/activemq-artemis-examples/" target="_parent">ActiveMQ Artemis Examples</a> repository.
+
+#### API
+
+*   [HTML](javadocs/javadoc-latest)
+
+See also the JMS / Jakarta Messaging API used for the respective Artemis clients:
+*   <a href="https://jakarta.ee/specifications/messaging/2.0/apidocs/" target="_parent">Jakarta Messaging 2.0 - javax.jms</a>
+*   <a href="https://jakarta.ee/specifications/messaging/3.1/apidocs/" target="_parent">Jakarta Messaging 3.1 - jakarta.jms</a>
```

Comment:
The various `target="_parent"` could be dropped now, it wouldn't have a parent anymore. Cant say I recall whether there is any issue with any leaving them or not, but they should be superfluous if not.

Reply:
Done.